twingate/internal/attr/user.go

```diff
@@ -1,10 +1,12 @@
 package attr
 
 const (
-	FirstName = "first_name"
-	LastName  = "last_name"
-	Email     = "email"
-	IsAdmin   = "is_admin"
-	Role      = "role"
-	Users     = "users"
+	FirstName  = "first_name"
+	LastName   = "last_name"
+	Email      = "email"
+	IsAdmin    = "is_admin"
```

Comment:
Why do we need this if we have `role` ?

Reply:
not sure, maybe for easy filtering when we use datasource
in the first implementation that we have for user datasource from 16/08/2022 we already have this two fields

Reply:
found this commit https://github.com/Twingate/terraform-provider-twingate/commit/81807422cac74a92cb61107f83e388bfb8e5b6a6